### DISCUSSION TITLE
Within the list that is fed into robot.directlyMoveTo, what exactly are the values? What do they correspond to? Is there documentation anywhere?

### DISCUSSION THREAD

**Discussion by molegod** [empriselab/RCareWorld#60] [Help - Programming] [answered]
![image](https://github.com/empriselab/RCareWorld/assets/19993701/5460b1b1-ca58-401b-9eb9-eaaf91d7a8eb)

For example, this is the initial position given, however, we are unable to determine what these values correspond to for the specific robot used in the provided 'test_dressing.py' file. Thank you!

**Comment by YoruCathy** [comment, marked as answer]
It’s documented as 
### **`directlyMoveTo(targetPose: list, targetRot: list = None)`**

Moves the robot to a target pose directly.

Args:

- **`targetPose`** (list): The target pose to move the robot to.
- **`targetRot`** (list): The target rotation (optional).

so the position is essentially a list with x y z elements for the robot to move its end effector to. The values in the starter script doesn’t matter because those are just the values of some random number to give you an idea about what to put as the parameters for the function. You need to figure out how to get that values. You can use the cube in the scene to help you visualize.
